imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,90 @@
+from django.test import TestCase
+from .models import Photo, Album, User
+from imager_profile.tests import UserFactory
+import factory
+
+
+class PhotoFactory(factory.django.DjangoModelFactory):
+    """Factory that creates fake photos for testing"""
+    class Meta:
+        model = Photo
+    title = factory.Faker('text', max_nb_chars=100)
+    description = factory.Faker('text')
+
+
+class AlbumFactory (factory.django.DjangoModelFactory):
+    """Factory that creates fake albums for testing"""
+    class Meta:
+        model = Album
+    title = factory.Faker('text', max_nb_chars=100)
+    description = factory.Faker('text')
+
+
+class PhotoAlbumUnitTests(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestCase, cls)
+        for _ in range(5):
+            user = UserFactory.create()
+            # user = User.objects.create(username='testuser')
+            user.set_password('CodeFellows')
+            user.save()    
+
+            album = AlbumFactory.create(user=user)
+            album.save()
+
+            photo = PhotoFactory.create(user=user)
+            photo.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        """tears down test objects"""
+        super(TestCase, cls)
+        Photo.objects.all().delete()
+        Album.objects.all().delete()
+
+    def test_photo_is_created(self):
+        """tests if photos were created successfully"""
+        self.assertTrue(Photo.objects.count())
+
+    def test_album_is_created(self):
+        """tests if album were created successfully"""
+        self.assertTrue(Album.objects.count()) 
+
+    def test_album_has_multiple_photos_and_users(self):
+        """tests if album takes multiple photos owned by multiple users"""    
+        album = Album.objects.first()    
+        album.save()
+        photo1 = Photo.objects.first()
+        photo2 = Photo.objects.all()[1]
+        photo1.user = User.objects.first()
+        photo2.user = User.objects.all()[1]
```

Comment:
You do not need to reassign the `user` like this. When you created the `Photo`s in the `setUpClass` each one was assigned a unique `User`.